tests/acquia_lift.test

```diff
@@ -2519,6 +2520,45 @@ class AcquiaLiftWebTestCampaignWizard extends AcquiaLiftWebTestBase {
     $this->assertOptionSelected('edit-audiences-everyone-else-assignment', $first_option_set->options[0]['option_id']);
   }
 
+
+  /**
+   * Test doing a straight test, i.e. all variations assigned to "Everyone Else".
+   */
+  function testNoTargeting() {
```

Comment:
I actually added this when I was doing this fix: https://github.com/acquia/personalize_module/commit/22ae8c8fcafb839427da576a27767a5bc9bdcfef Figured it was good to have a test for when you step through the wizard in the most basic way possible to ensure that things happens when they're supposed to happen.